package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -263,14 +268,20 @@ def guess_hydrogens(self, selection='all', max_mass=1.1, min_charge=0.3):
         Alternatively, this function may be used to quickly generate a :class:`str` of potential hydrogen atoms involved
         in hydrogen bonding. This str may then be modified before being used to set the attribute
         :attr:`hydrogens_sel`.
+
+        .. versionchanged: 1.0.0
+            Added `min_mass` parameter to specify minimum mass (Issue #2472)
         """
 
+        assert min_mass < max_mass
```

Comment:
I think we probably don't want assertionerrors in code, maybe valueerror is required here.